assets/src/js/store-lists.js

```diff
@@ -303,6 +309,17 @@
             });
         },
 
+        /**
+         * Get query param
+         * 
+         * @return boolean
```

Comment:
Kindly follow this guideline if you are using docblock for JS scripts.
https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/